### PR TITLE
Cleanup Query documentation

### PR DESCRIPTION
**Pull request by thePanz**
Cleanup Query documentation for ES-7
- [x] add `): self` to fluent methods
- [x] remove un-needed `@return $this` annotations
- [x] Updated changelog for breaking changes

### CODE REVIEW

File: lib/Elastica/Query.php [hunk 1]
```diff
@@ -23,16 +23,16 @@
...
+     * @var bool Suggest
      */
-    protected $_suggest = 0;
+    protected $hasSuggest = false;
 
     /**
      * Creates a query object.
      *
-     * @param array|\Elastica\Query\AbstractQuery $query OPTIONAL Query object (default = null)
+     * @param array|AbstractQuery $query Query object (default = null)
```
Comment (ruflin) [inline lib/Elastica/Query.php]: I had in the past the "Optional" inside to make it more obvious that there is a default when not looking at the code. But it should be obvious enough?

File: lib/Elastica/Query.php [hunk 2]
```diff
@@ -194,51 +171,45 @@ public function setHighlight(array $highlightArgs)
...
         return $this->addParam('highlight', $highlight);
     }
 
     /**
      * Sets maximum number of results for this query.
      *
-     * @param int $size OPTIONAL Maximal number of results for query (default = 10)
+     * @param int $size Maximal number of results for query (default = 10)
      *
-     * @return $this
```
Comment (ruflin) [inline lib/Elastica/Query.php]: Why do you consider the `return $this` as not needed anymore?

File: lib/Elastica/Query.php [hunk 3]
```diff
@@ -23,16 +23,16 @@
...
 {
     /**
-     * Suggest query or not.
+     * If the current query has a suggest in it.
      *
-     * @var int Suggest
+     * @var bool Suggest
      */
-    protected $_suggest = 0;
+    protected $hasSuggest = false;
```
Comment (ruflin) [inline lib/Elastica/Query.php]: Is it time to get rid of the protected `_` convention everywhere? The simple answer is yes, but how do we migrate there? Do we just adjust the new code to be different or all at once?

File: lib/Elastica/Query.php (hunk 1 above)
Comment (thePanz) [reply lib/Elastica/Query.php]: Yes, it is obvious as the `$var = null` signature :)

File: lib/Elastica/Query.php (hunk 2 above)
Comment (thePanz) [reply lib/Elastica/Query.php]: The method signature is `method(): self`, the annotation is no superfluous.
I am planning to take all of those into consideration in a next PR (php-cs-fixer can help here ;-) )

File: lib/Elastica/Query.php (hunk 3 above)
Comment (thePanz) [reply lib/Elastica/Query.php]: Yes, I'd move all of those to proper camelCase and get rid of the `_` prefix for protected properties in a next PR

Comment (ruflin) [reply lib/Elastica/Query.php]: I hear breaking change? Should we try to do it before 7.0 release?

**Comment by ruflin** [comment]
Seems CI is not happy :-(

File: lib/Elastica/Query.php (hunk 3 above)
Comment (thePanz) [reply lib/Elastica/Query.php]: Done, changelog updated too

**Comment by ruflin** [comment]
Thanks!

**Pull request merged**